### PR TITLE
Updating OVPsim device configuration and coverage processing

### PR DESCRIPTION
**Pull request by eroom1966**
Updated OVPsim log processing for coverage
Added OVPsim config files as targets

### CODE REVIEW

File: target/rv32imc/riscvOVPsim.ic [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# riscOVPsim configuration file converted from YAML
+--variant RVB32I
```
Comment (taoliug) [inline target/rv32imc/riscvOVPsim.ic]: Does this include RV32 M,C extension?

File: scripts/ovpsim_log_to_trace_csv.py [hunk 2]
```diff
@@ -113,7 +117,7 @@ def process_ovpsim_sim_log(ovpsim_log, csv, full_trace = 0):
         prev_trace.binary = trace_bin
         prev_trace.addr = trace_addr
         #prev_trace.privileged_mode = trace_mode
-        prev_trace.instr = trace_instr
+        prev_trace.instr = trace_instr # wrong
```
Comment (taoliug) [inline scripts/ovpsim_log_to_trace_csv.py]: What's the meaning of this comment?

Comment (simon5656) [reply scripts/ovpsim_log_to_trace_csv.py]: I feel there is some logic wrong in this section and this line '# wrong' might be superflous - but until we have many more regression tests I am just highlighting that this needs looking at in future maybe it needs a TODO - to investigate not 'wrong' - pls just decde

File: target/rv32imc/riscvOVPsim.ic (hunk 1 above)
Comment (simon5656) [reply target/rv32imc/riscvOVPsim.ic]: the target/rv32imc/riscvOVPsim.ic is derived from the riscv-config yaml specs that defines all legal options that a device can have and still meet the riscv specs.

in the riscvOVPsim.ic files there are several important statements that define the extensions. these are all documented in the riscvOVPsim docs, and also if you use , eg
   cd riscv-ovpsim/bin/Linux64
    riscvOVPsim.exe --showoverrides.

--variant RVB32I   # says this is a base rv32i - ie no U no S etc...  - the simplest.
--override riscvOVPsim/cpu/add_Extensions=MC # says add on M and C extensions
--override riscvOVPsim/cpu/misa_MXL=1 # and just confirm 32bits

so yes, to answer your question, Does this include RV32 M,C extension? it does.

thanks
Simon

File: scripts/ovpsim_log_to_trace_csv.py (hunk 2 above)
Comment (taoliug) [reply scripts/ovpsim_log_to_trace_csv.py]: OK, got it. I have fixed a few issues over the last couple of days. You can try to simulate with top of the tree to see if the issue is gone.

**Pull request merged**